circle.yml

```diff
@@ -7,40 +7,41 @@ machine:
   hosts:
     testpilot.dev: 127.0.0.1
   node:
-    version: 6.2.0
+    version: 6.9.1
   environment:
     TESTPILOT_ENABLE_PONTOON_BRANCHES: master
 
 dependencies:
 
   cache_directories:
     - "addon/node_modules"
-    - "node_modules"
-
-  pre:
-    # https://discuss.circleci.com/t/nvm-commands-are-not-evaluated/943
-    - nvm use 6.2.0 && nvm alias default 6.2.0
+    # node_modules is included by default
 
   override:
-    - node --version
```

Comment:
I think we do still want these versions included in the test output, to be clear about what we're using